pkg/user/api/validation/validation.go

```diff
@@ -73,6 +73,30 @@ func ValidateIdentityProviderUserName(name string) (bool, string) {
 	return ValidateUserName(name, false)
 }
 
+func ValidateGroup(group *api.Group) fielderrors.ValidationErrorList {
```

Comment:
> Shouldn't we validate group names as agreed? Check if they contain any colon?

Done in the `ValidateObjectMeta` via `ValidateGroupName` (https://github.com/openshift/origin/pull/3909/files#diff-4ae42dfcb3076ea0249403f62a41774cR46)